src/test/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/integration/service/PrisonOffenderEventListenerIntTest.kt

```diff
@@ -24,6 +24,8 @@ import java.time.Instant
 import java.time.LocalDateTime
 
 class PrisonOffenderEventListenerIntTest : SqsIntegrationTestBase() {
+  private val awaitAtMost30Secs
+    get() = await.atMost(Duration.ofSeconds(30))
```

Comment:
this funny setup is how `await` is defined, so following their example
https://github.com/awaitility/awaitility/blob/a56dce91b2abff03beb4773180ec6e5edb0117d7/awaitility-kotlin/src/main/kotlin/org/awaitility/kotlin/AwaitilityKt.kt#L29-L34